docs/advanced_topics/images/changing_rich_text_representation.md

```diff
@@ -25,7 +25,7 @@ class SubclassedImageFormat(Format):
 
 
 register_image_format(
-    SubclassedImageFormat('subclassed_format', 'Subclassed Format', classnames, filter_spec)
+    SubclassedImageFormat('subclassed_format', 'Subclassed Format', 'image-classes object-contain', filter_spec)
```

Comment:
Just updated the example code to be more 'real world', instead of some uncleared variable `classname`.